tags:
  {
   'tag_1': {size:  0.17, x: 2.08, y: 1.37, z: 0.7, qw: 1, qx: 0.0, qy: -1.5708, qz: -1.5708},
   'tag_3': {size:  0.17, x: 2.06, y: -0.68, z: 0.70, qw: 1, qx: 0.0, qy: -1.5708, qz: -1.5708},
   'tag_5': {size:  0.17, x: 2.25, y: 0.00, z: 1.49, qw: 1, qx: 0.0, qy: -1.5708, qz: -1.5708},
   'tag_7': {size:  0.17, x: 2.06, y: -0.68, z: 1.73, qw: 1, qx: 0.0, qy: -1.5708, qz: -1.5708},
   'tag_8': {size:  0.17, x: 2.06, y: 1.37, z: 1.76, qw: 1, qx: 0.0, qy: -1.5708, qz: -1.5708},
   'tag_9': {size:  0.17, x: -0.05, y: -1.63, z: 1.68, qw: 1, qx: 1.5708 , qy: 3.141592, qz: 0},
   'tag_10': {size:  0.17, x: 0.71, y: -1.62, z: 1.095, qw: 1, qx: 1.5708 , qy: 3.141592, qz: 0},
   'tag_11': {size:  0.17, x: 1.46, y: -1.6, z: 0.57, qw: 1, qx: 1.5708 , qy: 3.141592, qz: 0},
   'tag_12': {size:  0.17, x: 1.61, y: -1.60, z: 1.623, qw: 1, qx: 1.5708 , qy: 3.141592, qz: 0},
   'tag_13': {size:  0.17, x: 2.08, y: -1.37, z: 0.235, qw: 1, qx: 0.0, qy: -1.5708, qz: -1.5708},
   'tag_15': {size:  0.17, x: 0.46, y: 2.34, z: 0.24, qw: 1, qx: -1.57, qy: 3.14, qz: -3.14},
   'tag_16': {size:  0.17, x: 0.11, y: 2.32, z: 0.5, qw: 1, qx: -1.57, qy: 3.14, qz: -3.14},
   'tag_17': {size:  0.17, x: 1.03, y: 2.36 , z: 0.513, qw: 1, qx: -1.57, qy: 3.14, qz: -3.14},
   'tag_18': {size:  0.17, x: -0.145, y: -1.63, z: 0.745, qw: 1, qx: 1.5708 , qy: 3.141592, qz: 0},
  }

